### PR TITLE
custom branch configs

### PR DESCRIPTION
**Pull request by matt-winkler**
## Description & motivation

A few customers have had issues with this setting in dbt cloud repos, so thinking this will help remind SA's/CSE's to check this configuration during account setup.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md [hunk 1]
```diff
@@ -79,3 +79,11 @@ https://console.aws.amazon.com/codesuite/codecommit/repositories/<repo>/pull-req
 ```
 https://dev.azure.com/<org>/<repo>/_git/dbt/pullrequestcreate?sourceRef={{source}}&targetRef={{destination}}
 ```
+
+### Configuring Custom Branches on Repositories
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md]: If you make this `##` then the table of contents is going to look more sensible. Right now, this is rendering under the `Example templates` section.

I might actually recommend moving it up above the `Example templates` section with an `h2` altogether?

**Review by drewbanin — COMMENTED** [review]
One bit of feedback, but LGTM once addressed!

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md [hunk 2]
```diff
@@ -44,6 +44,14 @@ https://github.com/dbt-labs/jaffle_shop/compare/master..my-branch
 </TabItem>
 </Tabs>
 
+## Configuring Custom Branches on Repositories
+
+By default in Development Environments, dbt Cloud attempts to reference the `main` branch in connected repositories. If you want to use a different default branch name, you can configure dbt Cloud with a custom branch setting.
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md:49]: Is it still the case that the main branch must be named `main` or `master`?

Comment (matt-winkler) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md:49]: @amychen1776 dbt Cloud doesn't care what the base branch's name is.

Comment (amychen1776) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md:49]: Ah epic, as long as it's set as the default? That's cool

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "Configuring repositories"
 id: "cloud-configuring-repositories"
 ---
 
-### Configuring PR template URLs
+### Configuring Pull Request (PR) template URLs
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md]:
```suggestion
## Configure pull request (PR) template URLs
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md [hunk 4]
```diff
@@ -44,6 +44,14 @@ https://github.com/dbt-labs/jaffle_shop/compare/master..my-branch
 </TabItem>
 </Tabs>
 
+## Configuring Custom Branches on Repositories
+
+By default in Development Environments, dbt Cloud attempts to reference the `main` branch in connected repositories. If you want to use a different default branch name, you can configure dbt Cloud with a custom branch setting. 
+
+For example, the environment below is configured to use the `develop` branch of the connected repository:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md]:
```suggestion
For example, you can use the `develop` branch of a connected repository. Edit an environment, then in "General settings" select  **Only run on a custom branch** , and in "Custom branch" type **develop** or the name of your custom branch.
```

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md [hunk 5]
```diff
@@ -44,6 +44,14 @@ https://github.com/dbt-labs/jaffle_shop/compare/master..my-branch
 </TabItem>
 </Tabs>
 
+## Configuring Custom Branches on Repositories
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md]: I think we can shorten this as this page refers to repos 
```suggestion
## Configure custom branches
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
I updated this for the new UI and will replace the image shortly! Let's get this merged into `next` so we can release with the new UI content!

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**